iogt/templates/base.html

```diff
@@ -22,13 +22,24 @@
         <meta property="og:image" content="{% if self.social_media_image %}{{request.site.root_url}}{{ tmp_photo.url }}{% endif %}"/>
         <meta property="og:image:url" content="{% if self.social_media_image %}{{request.site.root_url}}{{ tmp_photo.url }}{% endif %}"/>
         <meta name="viewport" content="width=device-width, initial-scale=1" />
+        {% get_current_language as LANGUAGE_CODE %}
         {% compress css %}
-        <link rel="stylesheet" type="text/x-scss" href="{% static 'css/style.scss' %}">
+<<<<<<< HEAD
++           <link rel="stylesheet" type="text/x-scss" href="{% static 'css/style.scss'%}">
+           {% if LANGUAGE_CODE|language_bidi == True %}
+             <link rel="stylesheet" type="text/x-scss" href="{% static 'css/rtl-app.scss'%}">
+           {% endif %}
+=======
+            <link rel="stylesheet" type="text/x-scss" href="{% static 'css/style.scss'%}">
+          {% if LANGUAGE_CODE|language_bidi == True %}
+            <link rel="stylesheet" type="text/x-scss" href="{% static 'css/rtl-app.scss'%}">
```

Comment:
shouldn't the majority of the CSS changes be in this file? `rtl-app.scss?` also, can we rename this to `style_rtl.scss`